src/line-graph.js

```diff
@@ -113,6 +113,9 @@ export class LineGraph extends HTMLElement {
     var height = parseInt(this.dataset.height);
     var width = parseInt(this.dataset.width);
     var unit = this.dataset.unit;
+    if(unit === 'n') {
```

Comment:
Let's put this login in the `supportedInfo.js` file. If the unit of measurement is not found we will just return blank

Reply:
done with other logic!